sdk/python/feast/field.py

```diff
@@ -46,6 +65,252 @@ def dtype(self) -> ValueType:
         """
         return self._dtype
 
+    @property
+    def presence(self) -> schema_pb2.FeaturePresence:
```

Comment:
Please add comments to these properties.

Reply:
Ok